service-base/src/main/java/org/eclipse/hono/service/management/tenant/ResourceLimits.java

```diff
@@ -48,4 +49,20 @@ public void setExtensions(final Map<String, Object> extensions) {
     public Map<String, Object> getExtensions() {
         return this.extensions;
     }
+
+    /**
+     * Add a new extension entry.
```

Comment:
IMHO this should be changed to
> Adds or replaces an extension property.

to also reflect the *put* characteristics ...